spec/helpers/passengers_helper_spec.rb

```diff
@@ -12,7 +12,7 @@
 #     end
 #   end
 # end
-describe PassengersHelper do
+RSpec.describe PassengersHelper do
```

Comment:
Is this the monkey patching?

Reply:
Yes. RSpec monkey-patches `Object` to include a `.describe` method. But they kinda regret it.

Reply:
Meanwhile I'm over here in my other apps *coughumaps-railscough* removing every `Rspec` I see because I'm not used to seeing it and it seems wrong